bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -189,7 +189,10 @@ async def apply_infraction(
                 f"Infraction #{id_} actor is bot; including the reason in the confirmation message."
             )
             if reason:
-                end_msg = f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
+                end_msg = (
+                    f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})"
```

Comment:
```suggestion
                    f" (reason: {textwrap.shorten(reason, width=1500, placeholder='...')})."
```
Maybe?

Reply:
Can we add 2 line breaks between the 2 lines?

Reply:
Added period, newlines and non-mentioned moderator role in e68868e

![img](https://cdn.ionite.io/img/sPTEfL.jpg)